OpenCPPCoverage.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
These changes should hopefully clear up the missing/oversize exes.

Reply:
To summarize when building the x64 platform, don't build x86 (and vice versa); in Any CPU, build x64 and x86 binaries' projects.